src/MongoDB.Driver.Core/Core/Operations/ChangeStreamCursor.cs

```diff
@@ -25,20 +25,41 @@
 
 namespace MongoDB.Driver.Core.Operations
 {
+    /// <summary>
+    /// Represents an asynchronous cursor for change stream.
+    /// </summary>
+    /// <typeparam name="TDocument"></typeparam>
+    public interface IChangeStreamCursor<out TDocument> : IAsyncCursor<TDocument>
```

Comment:
It might make more sense to put this interface declaration in IAsyncCursor.cs.

Reply:
Done